Prolog/models/bird.pl

```diff
@@ -1,3 +1,32 @@
-:- module(bird,[testMethodBird/1]).
-
-testMethodBird(Param):- write(Param).
\ No newline at end of file
+:- module(bird,[create/4]).
+
+:- use_module(area).
+
+create(OriginX, OriginY, VerticalSpeed, Bird) :-
+    Bird=bird(originX(OriginX), originY(OriginY), verticalSpeed(VerticalSpeed)).
+
+originX(Bird, OriginX) :-
+    Bird=bird(originX(OriginX), originY(_), verticalSpeed(_)).
+
+originY(Bird, OriginY) :-
+    Bird=bird(originX(_), originY(OriginY), verticalSpeed(_)).
+
+verticalSpeed(Bird, VerticalSpeed) :-
+    Bird=bird(originX(_), originY(_), verticalSpeed(VerticalSpeed)).
+
+tick(Bird, NewBird) :-
+  originX(Bird, OriginX),
+  originY(Bird, OriginY),
+  verticalSpeed(Bird, VerticalSpeed),
+  NewOriginY is OriginY + VerticalSpeed,
+  NewVerticalSpeed is VerticalSpeed + 0.2,
```

Comment:
Do you think this `0.2` could be a parameter of this procedure (e.g. `Gravity`) and specified by the game controller?